yt_dlp/extractor/magentamusik.py

```diff
@@ -0,0 +1,41 @@
+from .common import InfoExtractor
+
+
+class MagentaMusikIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?magentamusik\.de/(?P<id>.*)'
+
+    _TESTS = [{
+        'url': 'https://www.magentamusik.de/marty-friedman-woa-2023-9208205928595409235',
+        'md5': 'd82dd4748f55fc91957094546aaf8584',
+        'info_dict': {
+            'id': '9208205928595409235',
+            'ext': 'mp4',
+            'title': 'Marty Friedman: W:O:A 2023',
+            'duration': 2760
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
```

Comment:
I would change this to be the `display_id` and also return it in the info dict.
```suggestion
        display_id = self._match_id(url)
        webpage = self._download_webpage(url, display_id )
```